packages/ai-jsx/readme.md

```diff
@@ -4,7 +4,9 @@
 [![Discord Follow](https://dcbadge.vercel.app/api/server/MsKAeKF8kU?style=flat)](https://discord.gg/MsKAeKF8kU)
 [![Twitter Follow](https://img.shields.io/twitter/follow/fixieai?style=social)](https://twitter.com/fixieai)
 
-[<img src="../docs/static/img/loom.png" alt="Image description" width="50%">](https://www.loom.com/share/c13b0c73f8d34f9f962048b39a4794f6?sid=a693f220-ccb1-4e70-913a-c2eda86dd0ce)
+## AI.JSX in Two Minutes
+
+https://github.com/fixie-ai/ai-jsx/assets/476553/301b79e4-7023-4adc-a3a5-72d5b7af0cde
```

Comment:
![image](https://github.com/fixie-ai/ai-jsx/assets/829827/2b3fcb2d-46cc-4128-bbfb-0f2158603c68)

This embeds very nicely in GH but I bet it'll just be a bare link on npm, which is the main place people will be reading this file.

Reply:
yes good point. will change this PR to stage in a test package first.

Reply:
investigating this...